pandas/tests/arithmetic/test_numeric.py

```diff
@@ -1297,3 +1297,10 @@ def test_dataframe_div_silenced():
     )
     with tm.assert_produces_warning(None):
         pdf1.div(pdf2, fill_value=0)
+
+
+def test_inplace_arithmetic_operation_on_series_updating_parent_dataframe():
```

Comment:
this test passes for me on master.  is there another case that fails?

Reply:
yes I think need to test `df["A"] is s` which is also True on 1.1.2

```
>>> pd.__version__
'1.1.2'
>>>
>>> import pandas.testing as tm
>>>
>>> df = pd.DataFrame({"A": [1, 2, 3]})
>>> s = df["A"]
>>> s += 1
>>> tm.assert_series_equal(df["A"], s)
>>>
>>> s
0    2
1    3
2    4
Name: A, dtype: int64
>>>
>>> df["A"]
0    2
1    3
2    4
Name: A, dtype: int64
>>>
>>> df
   A
0  1
1  2
2  3
>>> df["A"] is s
True
>>>
```

but the output of \_\_repr__ and also other ops is incorrect

so should also compare using assert_frame_equal

```
>>> tm.assert_frame_equal(df, pd.DataFrame({"A": [2, 3, 4]}))
Traceback (most recent call last):
...
AssertionError: DataFrame.iloc[:, 0] (column name="A") are different

DataFrame.iloc[:, 0] (column name="A") values are different (100.0 %)
[index]: [0, 1, 2]
[left]:  [1, 2, 3]
[right]: [2, 3, 4]
>>>
```
but this should pass with the fix here